package-lock.json

```diff
@@ -1449,11 +1449,11 @@
       }
     },
     "node_modules/braces": {
-      "version": "3.0.2",
-      "resolved": "https://registry.npmjs.org/braces/-/braces-3.0.2.tgz",
-      "integrity": "sha512-b8um+L1RzM3WDSzvhm6gIz1yfTbBt6YTlcEKAvsmqCZZFw46z626lVj9j1yEPW33H5H+lBQpZMP1k8l+78Ha0A==",
+      "version": "3.0.3",
```

Comment:
Why are we updating the `package-lock.json` when `package.json` hasn't changed?

Reply:
`npm audit fix` changed that to fix a vulnerability in one of the sub-dependencies (which are not listed in package.json)